README.md

```diff
@@ -9,7 +9,9 @@ Provided packages:
 * `github.com/elastic/elastic-agent-libs/cloudid` is used for parsing `cloud.id` and `cloud.auth` when connecting to the Elastic stack.
 * `github.com/elastic/elastic-agent-libs/config` the previous `config.go` file from `github.com/elastic/beats/v7/libbeat/common`. A minimal wrapper around `github.com/elastic/go-ucfg`. It contains helpers for merging and accessing configuration objects and flags.
 * `github.com/elastic/elastic-agent-libs/file` is responsible for rotating and writing input and output files.
+* `github.com/elastic/elastic-agent-libs/filewatcher` Watches files and notifies if they have been modified.
 * `github.com/elastic/elastic-agent-libs/keystore` interface for keystores and file keystore implementation.
+* `github.com/elastic/elastic-agent-libs/loader` Helpers for loading a main elastic-agent config file.
```

Comment:
Was this line moved on purpose? Seems weird that this is a part of the PR.

Reply:
I organised them alphabetically, I might have forgotten to mention this on the PR body. That was the only one out of place.